ambry-router/src/main/java/com/github/ambry/router/DeleteOperation.java

```diff
@@ -176,7 +176,9 @@ void handleResponse(ResponseInfo responseInfo, DeleteResponse deleteResponse) {
     if (responseInfo.getError() != null) {
       logger.trace("DeleteRequest with response correlationId {} timed out for replica {} ",
           deleteRequest.getCorrelationId(), replica.getDataNodeId());
-      onErrorResponse(replica, new RouterException("Operation timed out", RouterErrorCode.OperationTimedOut));
+      onErrorResponse(replica, new RouterException(
+          "Operation timed out because of " + responseInfo.getError() + " at DataNode " + responseInfo.getDataNode(),
+          RouterErrorCode.OperationTimedOut));
```

Comment:
`deleteRequest` should be able to get blob id, this may be worthwhile to log.
Same for replica id.

Reply:
you mean log out the blob id and replica id?

Reply:
yes

Reply:
replica id is the same ad datanode here so I just log out blob id.

Reply:
actually replicaId contains datanode info but datanode has no info about replica

Reply:
I mean we only want to know which host failed, datanode can provide that info.